components/next_steps_view/images/logo-svg.tsx

```diff
@@ -0,0 +1,31 @@
+                fill='#166DE0'
+            />
+            <path
+                fillRule='evenodd'
+                clipRule='evenodd'
+                d='M18.7948 11.4403C18.7948 11.4403 18.8382 13.3508 17.5134 14.6487C16.1884 15.9466 14.5611 15.828 13.5025 15.4695C12.4437 15.1111 11.079 14.2168 10.8149 12.3811C10.5508 10.5453 11.7458 9.05398 11.7458 9.05398L14.3497 5.81577L15.8663 3.96632L17.1681 2.35463C17.1681 2.35463 17.7655 1.55441 17.9358 1.38925C17.9695 1.35653 18.0041 1.33505 18.038 1.31852L18.0627 1.30591L18.0672 1.3042C18.1388 1.27335 18.2213 1.26671 18.3009 1.29364C18.3787 1.32006 18.4391 1.37357 18.4773 1.43953L18.4853 1.45232L18.4923 1.46698C18.5108 1.50209 18.5263 1.54214 18.5337 1.59157C18.5684 1.82626 18.557 2.82488 18.557 2.82488L18.6121 4.89573L18.6934 7.28616L18.7948 11.4403ZM22.1655 2.67214C26.9815 6.17725 29.1903 12.5383 27.1819 18.4877C24.7101 25.8095 16.7872 29.7357 9.48576 27.2572C2.18429 24.7785 -1.73091 16.8337 0.740758 9.51193C2.75249 3.55281 8.37502 -0.156665 14.3346 0.00508354L12.4243 2.26837C8.88883 2.90939 5.83622 5.33051 4.63642 8.88471C2.85125 14.1726 5.84335 19.9664 11.3195 21.8253C16.7956 23.6842 22.682 20.9045 24.4672 15.6166C25.6631 12.0742 24.7149 8.30504 22.3125 5.6436L22.1655 2.67214Z'
+                fill='#166DE0'
```

Comment:
I think you want this svg colors fills to use the variables so it can be use with multiple color themes.